homeassistant/components/frontend/version.py

```diff
@@ -1,2 +1,2 @@
 """DO NOT MODIFY. Auto-generated by build_frontend script."""
-VERSION = "7baf9a2bf8d94448eeaa6a1ceaf8464e"
+VERSION = "eb32bd03ce1653c03f36ae5dea80eb04"
```

Comment:
Please do not include the new version as changes have not been merged yet. I'll make sure they make it into dev when the other PR has been approved.